.github/workflows/github_actions.yml

```diff
@@ -0,0 +1,102 @@
+name: TileDB-Trino
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  Test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-18.04, macos-10.15 ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          unset SYSTEM
+          set -e pipefail
+          open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg
+          sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -allowUntrusted -target /
+        shell: bash
+        name: Install_system_headers_mac_only
+        if: matrix.os == 'macOS-10.14'
+
+      - run: |
+          unset SYSTEM
```

Comment:
Is this necessary? It was an issue w/ AZP, but I don't know if GHA does this.

Reply:
Do you refer to the same thing as above?

Reply:
Yes, nevermind